### ISSUE TITLE
Several interfaces implementing bug

### ISSUE THREAD

**Issue by NagorniyAleksandr** [tinnou/graphql-codegen#1]
Hi! Thanks a lot for your work. It's great code generator!

In my case some class is implementing several interfaces. Their schema is like:
```json
{
  "kind": "OBJECT",
  "name": "myClass",
  "description": "No description for myClass",
  "fields": [],
  "inputFields": null,
  "interfaces": [
    {
      "kind": "INTERFACE",
      "name": "firstInterface",
      "ofType": null
    },
    {
      "kind": "INTERFACE",
      "name": "secondInterface",
      "ofType": null
    }
  ],
  "enumValues": null,
  "possibleTypes": null
}
```
, so generated code should looks like:
```
public class MyClass implements FirstInterface, SecondInterface {
   ....
}

public interface FirstInterface {
   ....
}

public interface SecondInterface {
   ....
}
```

but the result of generating of the code is:

```
public class Appy_Deployment implements firstInterface,  implements secondInterface {
   ....
}

public interface FirstInterface {
   ....
}

public interface SecondInterface {
   ....
}
```
and we have:
1. duplicated  "implements"
2. chain of interface names witch starts from lower letter (when interface class name starts from Upper letter).
As a result we have compilation error.

I've already fixed this issue in my local copy, so I can push code with fixes to some branch if you grant me the appropriate permissions.

**Comment by tinnou** [comment]
Thank you for your feedback, I welcome contributions. Can you create a [pull request](https://help.github.com/articles/creating-a-pull-request/) with your proposed changes and a few tests covering them?

**Comment by tinnou** [comment]
Fixed by #3
